File: apps/s-2-admin/src/api/verificationToken/VerificationTokenWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { StringFilter } from "../../util/StringFilter";
import { DateTimeFilter } from "../../util/DateTimeFilter";

export type VerificationTokenWhereInput = {
  id?: IntFilter;
  identifier?: StringFilter;
  token?: StringFilter;
  expires?: DateTimeFilter;
  createdAt?: DateTimeFilter;
  updatedAt?: DateTimeFilter;
};

File: apps/cactus-demo/src/APIs/VerificationToken/Dtos/VerificationTokenWhereInput.cs
namespace CactusDemo.APIs.Dtos;

public class VerificationTokenWhereInput
{
    public int? Id { get; set; }

    public string? Identifier { get; set; }

    public string? Token { get; set; }

    public DateTime? Expires { get; set; }

    public DateTime? CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }
}
